src/audio.rs

```diff
@@ -551,11 +559,11 @@ fn floor_one_curve_synthesis(floor1_final_y :Vec<u32>,
 	} else if hx > n as u32 {
 		floor.truncate(n as usize);
 	}
-	let mut rf = Vec::with_capacity(floor.len());
-	for fl in floor {
-		rf.push(FLOOR1_INVERSE_DB_TABLE[fl as usize]);
-	}
-	return rf;
+
+	floor.into_iter()
+		.map(|fl| fl as usize)
```

Comment:
Could you fold this map into the next one? This would improve compile times, I hope :)